### DISCUSSION TITLE
Support for earlier IOS versions?

### DISCUSSION THREAD

**Discussion by pilotcatxd** [MaxiHuHe04/iTunes-Backup-Explorer#23] [Q&A]
Hi, really appreciate the work your team has done in bringing an accessible backup explorer in an era where Apple stops people from having access to their own files without pay. 

Any chance you'd roll out support for this to earlier versions of iOS in the future? 

I have a really old iPhone 4 backup for my mom that I'm trying to extract photos and chat backups (WhatsApp etc) from and can't find any way to do it currently :(

**Comment by d3ibis** [comment]
Same situation here, I have an app can open old backups but I'm not comfortable with it.

I have backups of iPhone 5s & 6 (iOS 8.4 & 9.0.2)

This is what I get when trying to load them:

![ice_video_20240824-191626](https://github.com/user-attachments/assets/1b4fa079-4c2a-4157-8153-d32c400ef797)

**Reply by DavidDBruce** [reply]
Same issue here, so it isn't just me. 

Looks like they went to using the SQLite DB file at some point which I wasn't aware of.

Hoping there would be a solution for this at some point.
